src/api/procedures/createGroup.ts

```diff
@@ -85,12 +85,16 @@ export async function prepareCreateGroup(
   const currentGroupPermissions = await P.map(groups, group => group.getPermissions());
 
   if (
-    currentGroupPermissions.some(({ transactions: transactionPermissions }) =>
-      isEqual(
-        optionize(orderTransactionPermissionsValues)(transactionPermissions),
-        optionize(orderTransactionPermissionsValues)(transactions)
-      )
-    )
+    currentGroupPermissions.some(({ transactions: transactionPermissions }) => {
+      if (transactionPermissions) {
+        transactionPermissions.values.sort();
+      }
+      if (transactions) {
+        transactions.values.sort();
+      }
+
+      return isEqual(transactionPermissions, transactions);
```

Comment:
```suggestion
      const sortedTransactions: TransactionPermissions | null = 
        transactions && {
          ...transactions,
          values: [...transactions.values].sort(),
        };

      return isEqual(transactionPermissions, sortedTransactions);
```

In this way, you aren't mutating the original object